scripts/live_test/CLITest.yml

```diff
@@ -375,17 +375,20 @@ jobs:
         az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
         az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
 
+        # Clean resources
+        python /home/vsts/work/1/s/scripts/live_test/clean_policy.py
```

Comment:
Can you use relative location?

Reply:
Yes. But we must pay attention to what current pwd is.